scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -15,7 +15,7 @@ workflow GvsExtractCallset {
 
     File interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
     File interval_weights_bed = "gs://broad-public-datasets/gvs/weights/gvs_vet_weights_1kb.bed"
-    File gatk_override = "gs://broad-dsp-spec-ops/scratch/bigquery-jointcalling/jars/rc-add-AD-04112022/gatk-package-4.2.0.0-498-g1f53709-SNAPSHOT-local.jar"
+    File gatk_override = "gs://broad-dsp-spec-ops/scratch/bigquery-jointcalling/jars/rc-update-headers-04142022/gatk-package-4.2.0.0-497-gda8a97a-SNAPSHOT-local.jar"
```

Comment:
Should these directory have the jira ticket number embedded (to see if the work has been merged into ah_var_store and so we can assume the directory and jar is stale / unneeded).

Reply:
Yeah I agree that would be good. The combo of embedding the ticket number in the branch name and the "copy your custom GATK jar" script that Kristian shared the other day we seem to get that.

Reply:
I like this! Maybe post-Charlie we can consolidate a set of rules for jars? It's certainly too haphazard now